src/App.js

```diff
@@ -1,23 +1,107 @@
-import logo from './logo.svg';
+import axios from 'axios';
```

Comment:
Your code was very well organized and easy to follow along!  You made a great app and the code side of it is awesome too.  From my view, your code looks DRY and easy to understand!